README.rst

```diff
@@ -63,14 +63,69 @@ Publishing
     git tag 0.5.1
     git push origin 0.5.1
 
-4. Register the package::
+``.pypirc`` file
+-----------------
+
+To ensure publication and testing of publications are submitted to the correct
```

Comment:
I would just skip the explanation and say "Configure your `.pypirc` file for your production and test server accounts. This file should look as follows:", by the time you follow through the rest of the guide, it should be apparent what this is for. That's my two cents.